index.js

```diff
@@ -1,4 +1,4 @@
-import WebView from './lib/WebView';
+import WebView from './src/WebView';
```

Comment:
Can we remove this from this PR? It is an issue just should be in a separate PR.